Bio/SearchIO/HmmerIO/hmmer3_domtab.py

```diff
@@ -296,7 +294,7 @@ def _build_row(self, qresult):
                 qaccw, qresult_acc, qresult.seq_len, hit.evalue, hit.bitscore,
                 hit.bias, hsp.domain_index, len(hit.hsps), hsp.evalue_cond, hsp.evalue,
                 hsp.bitscore, hsp.bias, hmm_from, hmm_to, ali_from, ali_to,
-                hsp.env_start + 1, hsp.env_end, hsp.acc_avg, hit.description)
+               hsp.env_start + 1, hsp.env_end, hsp.acc_avg, hit.description)
```

Comment:
This looks like an accidentally committed typo (removing a space and breaking the indentation alignment)